infrastructure/validators/rewards-payout.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Rewards Payout
+description: Learn how validator rewards work on the network, including era points, payout distribution, running multiple validators, and nominator payments.
+---
+# Rewards Payout
+
+## Introduction
+
+Understanding how rewards are distributed to validators and nominators is essential for network participants. In Polkadot and Kusama, validators earn rewards based on their era points, which are accrued through actions like block production and parachain validation. This guide explains the payout scheme, factors influencing rewards, and how multiple validators affect returns. Validators can also share rewards with nominators, who contribute by staking behind them. By following the payout mechanics, validators can optimize their earnings and better engage with their nominators.
+
+## Era Points
+
+For every era (a period of time approximately 6 hours in length in Kusama, and 24 hours in Polkadot), validators are paid proportionally to the amount of [era points](TODO: path) they have collected. Era points are reward points earned for payable actions like:
+
+- Issuing validity statements for [parachain](TODO: path) blocks
+- Producing a non-uncle block in the relay chain
+- Producing a reference to a previously unreferenced uncle block
+- Producing a referenced uncle block
+
+!!!note
+    An uncle block is a Relay Chain block that is valid in every regard, but which failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. We call the lagging blocks uncle blocks.
```

Comment:
```suggestion
    An uncle block is a Relay Chain block that is valid in every regard but has failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. The lagging blocks are called uncle blocks.
```